recipes/igrafx-mining-sdk/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "igrafx-mining-sdk" %}
+{% set version = "2.28.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/igrafx_mining_sdk-{{ version }}.tar.gz
+  sha256: cd1a162515ba7b595c08da3119522408864f2b3ddb61884ef73fb3873534d1c6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python ==3.9
+    - pip
+    - poetry ==1.7.1
+  host:
+    - pip
+    - python ==3.9
+    - poetry ==1.7.1
+    - SQLAlchemy ==2.0.12
+  run:
+    - python ==3.9
+    - poetry ==1.7.1
+    - requests ==2.29.0
+    - pydruid ==0.6.5
+    - pandas ==1.3.5
+    - SQLAlchemy ==2.0.12
+    - networkx ==2.6.3
+    - numpy ==1.21.6
+    - toml ==0.10.2
+    - python-dotenv ==0.19.2
+    - pytest ==7.3.1
+    - pytest-dependency ==0.5.1
```

Comment:
Can you please update these dependencies according to https://github.com/igrafx/mining-python-sdk/blob/7ef88c2e9f11bcfe1334d03ddcefbffb12e5a639/pyproject.toml#L12 and not make them as strict as you currently have it?